### PR TITLE
Main screen with Tab layout, search bar

### PR DESCRIPTION
**Pull request by nvs266**

### CODE REVIEW

File: app/src/main/java/com/framgia/mysoundcloud/screen/main/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    }
+
+    private void initializeToolbar() {
+        Toolbar toolbar = findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        updateTitle(getString(R.string.music_genres));
+    }
+
+    private void initializeSearchView(Menu menu) {
+        MenuItem searchItem = menu.findItem(R.id.action_search);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/mysoundcloud/screen/main/MainActivity.java]: rename to menuSearch or searchMenu look better

File: app/src/main/java/com/framgia/mysoundcloud/screen/main/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        if (searchItem != null) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/mysoundcloud/screen/main/MainActivity.java]: can you if (searchItem ==null) {
return;
}

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -1,8 +1,19 @@
...
+    <color name="colorAccent">#FF5722</color>
+    <color name="colorPrimaryText">#212121</color>
+    <color name="colorSecondaryText">#757575</color>
+    <color name="colorDivider">#BDBDBD</color>
+
+    <!--Orange-->
+    <color name="deep_orange_500">#E64A19</color>
+
+    <!--Gray-->
+    <color name="gray_01">#E8E8E8</color>
```
Comment (DoanVanToan) [inline app/src/main/res/values/colors.xml]: e có thể đặt tên color theo mẫu sau
http://www.color-blindness.com/color-name-hue/
resource name đặt tên lower_case_under_score

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,4 +1,10 @@
 <resources>
     <string name="app_name">My SoundCloud</string>
     <string name="msg_hello_world">Hello World!</string>
+    <string name="search_title">Search tracks</string>
+    <string name="music_genres">Music genres</string>
+    <string name="msg_finding_tracks">Finding tracks...</string>
+    <string name="msg_feature_is_coming">Feature is coming...</string>
+    <string name="msg_opps">Opps!</string>
+    <string name="download">Download</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml]: rename to action_download

File: app/src/main/java/com/framgia/mysoundcloud/screen/main/MainActivity.java (hunk 1 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/mysoundcloud/screen/main/MainActivity.java]: solved

File: app/src/main/java/com/framgia/mysoundcloud/screen/main/MainActivity.java (hunk 2 above)
Comment (nvs266) [reply app/src/main/java/com/framgia/mysoundcloud/screen/main/MainActivity.java]: solved

File: app/src/main/res/values/colors.xml (hunk 3 above)
Comment (nvs266) [reply app/src/main/res/values/colors.xml]: solved

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (nvs266) [reply app/src/main/res/values/strings.xml]: solved

**Pull request merged**